extensions/wikia/Insights/InsightsHooks.class.php

```diff
@@ -60,4 +60,18 @@ public static function onGetLocalURL( &$this, &$url, $query ) {
 
 		return true;
 	}
+
+	/**
+	 * Disable create new page popup
```

Comment:
I think we can be more explicit here as well and explain why we're disabling popup on insights page.